pkg/rulesConfig/defaultRules/github/ensure-github-action-version.json

```diff
@@ -0,0 +1,50 @@
+{
+    "description": "Ensure action version is set",
+    "failureMessage": "github action version should be set",
+    "schema": {
+        "type": "object",
+        "additionalProperties": {
+            "type": "object",
+            "properties": {
+                "repositories": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "object",
+                        "properties": {
+                            "github-actions-workflows": {
+                                "type": "object",
+                                "additionalProperties": {
+                                    "properties": {
+                                        "content": {
+                                            "properties": {
+                                                "jobs": {
+                                                    "type": "object",
+                                                    "additionalProperties": {
+                                                        "properties": {
+                                                            "steps": {
+                                                                "type": "array",
+                                                                "items": {
+                                                                    "type": "object",
+                                                                    "properties": {
+                                                                        "uses": {
+                                                                            "type": "string",
+                                                                            "pattern": ".*@.*"
```

Comment:
What happens if I do `uses: "path/to/action@"`? I don't know the behavior of github in this case (maybe it's okay and it still takes the latest? I don't know), but maybe we should make sure there's something after the `@`? this would mean to change the rule to `.*@.+`